include/model.php

```diff
@@ -757,6 +745,7 @@ protected function update_secondary_language_terms( $slug, $name, PLL_Language $
 				continue;
 			}
 
+			/** @var PLL_Language $language */
```

Comment:
`$language instanceof PLL_Language` or `! empty( $language )` is better, isn't?

Reply:
We know it is an instance of `PLL_Language` because we do this (see the inline comments):
```php
if ( ! empty( $language ) ) {
	// $language is an instance of PLL_Language.
	$term_id = $language->get_tax_prop( $object->get_tax_language(), 'term_id' );
} else {
	// $language is empty.
	$term_id = 0;
}

if ( empty( $term_id ) ) {
	// $language is empty, or get_tax_prop() returned an empty term_id (let's say it's possible).
	wp_insert_term( $name, $object->get_tax_language(), array( 'slug' => $slug ) );
	continue;
}

// At this point, $language cannot be empty.
```

So, we know that `$language` is an instance of `PLL_Language`, but PHPStan doesn't seem to understand it. So I preferred to add an inline comment for PHPStan instead of adding a useless `instanceof` or `empty()` test.
So what do you prefer?